.github/workflows/deploy.yml

```diff
@@ -522,8 +522,10 @@ jobs:
 
       - name: Install tools
         run: |
-          sudo apt-get update
-          sudo apt-get install -y virtualbox vagrant
+          wget -O- https://apt.releases.hashicorp.com/gpg | sudo gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg
```

Comment:
Without these we get `Package 'vagrant' has no installation candidate`.
https://developer.hashicorp.com/vagrant/downloads#linux